app/controllers/questions_controller.rb

```diff
@@ -0,0 +1,42 @@
+class QuestionsController < ApplicationController
+
+  def index
+    @questions = Question.all
+  end
+
+  def new
+    @question = Question.new
+  end
+
+  def create
+    @new_question = Question.new(question_params)
+    if @new_question.save!
+      redirect_to question_path(@new_question)
+    else
+      flash[:notice] = "A problem has occurred. Your question couldn't be posted."
+      redirect_to questions_path
+    end
+  end
+
+  def show
+    @question = Question.find(params[:id])
+    @question_vote = @question.votes

```

Comment:
Once you have your @question, its votes are available in the view as question.votes - you don't need to separately assign to question_vote.